sdks/java/io/kafka/build.gradle

```diff
@@ -65,26 +76,68 @@ dependencies {
   testCompile library.java.junit
   testCompile library.java.powermock
   testCompile library.java.powermock_mockito
+  testCompile library.java.testcontainers_kafka
   testRuntimeOnly library.java.slf4j_jdk14
   testRuntimeOnly project(path: ":runners:direct-java", configuration: "shadow")
-  kafkaVersion210 "org.apache.kafka:kafka-clients:2.1.0"
+  kafkaVersions.each {"kafkaVersion$it.key" "org.apache.kafka:kafka-clients:$it.value"}
 }
 
-configurations.kafkaVersion210 {
-  resolutionStrategy {
-    force "org.apache.kafka:kafka-clients:2.1.0"
+kafkaVersions.each { kv ->
+  configurations."kafkaVersion$kv.key" {
+    resolutionStrategy {
+      force "org.apache.kafka:kafka-clients:$kv.value"
+    }
   }
 }
 
-task kafkaVersion210Test(type: Test) {
```

Comment:
I think this task was used to run KafkaIO unit tests against specific client version. It's not a case now?

Reply:
This task is still there at
```
kafkaVersions.each {kv ->
  task "kafkaVersion${kv.key}Test"(type: Test)
  [...]
```
Though I used 2.1.1 version - should we leave 2.1.0?

Reply:
I think it should fine since 2.1.0 was used as a minimum version where headers were added (if I'm not mistaken).

Reply:
I'm not sure as well but I guess so. I'm having a difficulty with the recently added Kafka streaming performance tests so  I'm changing this PR to WIP.

If I understand correctly these streaming tests can't be run on DirectRunner? They never stop even with `readTimeout` set to 1min

Reply:
Kafka performance tests have been failing (timeout) since https://github.com/apache/beam/pull/13282 merge.

I guess this PR should wait until it's fixed?

https://ci-beam.apache.org/job/beam_PerformanceTests_Kafka_IO/

Reply:
@boyuanzz Could you take a look on https://ci-beam.apache.org/job/beam_PerformanceTests_Kafka_IO/1563/ ? After rebase with latest KafkaIOIT changes, It fails because of timeout.

Reply:
Looking.

Reply:
It seems like the sdk docker image is cleaned up incorrectly before test finishes. I can have a quick fix for that.

Reply:
So the problem is introduced by https://github.com/apache/beam/pull/13283/files

Reply:
@boyuanzz Thanks! I guess it's should fixed now (#13356 ), right?